src/client/chat.js

```diff
@@ -69,6 +69,13 @@ module.exports = function (client, options) {
     }
   })
 
+  client.on('system_chat', (packet) => {
+    client.emit('systemChat', {
```

Comment:
Can you document that event ?

Reply:
It is not a new event so should be documented already

Reply:
https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/docs/API.md seems it's not

Everything that is exposed / used by users should be documented so it's clear if we make breaking changes or not

Reply:
Interesting. Should have been included in the initial chat signing PR. 

Gonna add it to this one